File: app/src/main/java/com/scatl/uestcbbs/module/message/view/ReplyMeMsgView.java
package com.scatl.uestcbbs.module.message.view;

import com.scatl.uestcbbs.entity.ReplyMeMsgBean;

/**
 * author: sca_tl
 * description:
 * date: 2020/1/29 15:22
 */
public interface ReplyMeMsgView {
    void onGetReplyMeMsgSuccess(ReplyMeMsgBean replyMeMsgBean);
    void onGetReplyMeMsgError(String msg);
}

File: app/src/main/java/com/scatl/uestcbbs/module/message/view/ReplyMeMsgView.kt
package com.scatl.uestcbbs.module.message.view

import com.scatl.uestcbbs.base.BaseView
import com.scatl.uestcbbs.entity.ReplyMeMsgBean

/**
 * Created by sca_tl at 2023/2/17 10:05
 */
interface ReplyMeMsgView: BaseView {
    fun onGetReplyMeMsgSuccess(replyMeMsgBean: ReplyMeMsgBean)
    fun onGetReplyMeMsgError(msg: String?)
}